code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -182,6 +197,19 @@ There are important things regarding this file:
 	penetrating = 5
 	hitscan = TRUE //so the PTR isn't useless as a sniper weapon
 
+/obj/item/projectile/bullet/antim/s
+	damage_types = list(BRUTE = 63)
+
+/obj/item/projectile/bullet/antim/owo
```

Comment:
No.

Reply:
This has been approved by Reere 2.0, aka Firefox on the condition it's admin only.